examples/src/main/java/com/mapbox/navigation/examples/core/SimpleMapboxNavigationKt.kt

```diff
@@ -70,6 +70,11 @@ import kotlinx.coroutines.channels.Channel
 import okhttp3.Cache
 import timber.log.Timber
 
+/**
+ * This activity shows how to set up a basic turn-by-turn
+ * navigation experience with many of the the Navigation SDK's
```

Comment:
And it's called `SimpleMapboxNavigationKt` 😅 We should definitely rename this example.